src/httprpc.cpp

```diff
@@ -18,10 +18,23 @@
 #include <stdio.h>
 
 #include <boost/algorithm/string.hpp> // boost::trim
+#include <boost/signals2/signal.hpp>
 
 /** WWW-Authenticate to present with 401 Unauthorized response */
 static const char* WWW_AUTH_HEADER_DATA = "Basic realm=\"jsonrpc\"";
 
+boost::signals2::signal<void (JSONRPCRequest&, const HTTPRequest&)> PrepareJSONRPCRequestCallbacks;
```

Comment:
Isn't the `HTTPRequest` object unused? Why pack it into the signal, future extensions?

Reply:
Right